### PR TITLE
finished one of the functions in the diagnostic

### PR DESCRIPTION
**Pull request by gastevend**

### CODE REVIEW

**Comment by gastevend** [comment]
comfort: 4 
clarity: 5 
mySpeed: 0

I referenced the lab we did in class on this

File: lib/diagnostic.js [hunk 1]
```diff
@@ -6,16 +6,34 @@
...
 // and returns that array.
-const normalizeWords = function (/* */) {
-  /* your response here */
+const normalizeWords = function (words) {
+  let splitWords = words.split(' ')
+  let upperCase = [];
+  for (let i = 0; i < splitWords.length; i++) {
+     upperCase.push(splitWords[i].toUpperCase().replace(/[^A-Za-z0-9_]/g, ''))
+  }
+  return upperCase
```
Comment (benjimelito) [inline lib/diagnostic.js:15]: Cool 🕶

File: lib/diagnostic.js [hunk 2]
```diff
@@ -6,16 +6,34 @@
...
+     if (upperCase[splitWords[i]] !== undefined) {
+       upperCase[splitWords[i]]++
+     } else {
+       upperCase[splitWords[i]] = 1
+     }
+  }
+  let unique = []
+  for (key in )
+
+// This is all messed up need to come back to it when i have time. :(
```
Comment (benjimelito) [inline lib/diagnostic.js:33]: On the right track here...

**Comment by benjimelito** [comment]
Seems like you have a good handle on things, timed diagnostics are tough ☹️ . Even that being the case, please try to get these in before 9:30am (9:31am)

**Pull request closed without merging**